python/cuml/cuml/manifold/umap.pyx

```diff
@@ -510,15 +520,10 @@ class UMAP(UniversalBase,
             else:
                 umap_params.build_algo = graph_build_algo.NN_DESCENT
                 build_kwds = self.build_kwds or {}
```

Comment:
We _could_ add a warning in here for anyone passing `nnd_graph_degree` letting them know that it's now ignored. I opted to skip doing that, but happy to add it if asked.

Likewise, we _might_ want to error if there's any extra unknown parameters here, no strong thoughts.

Reply:
I think a warning/error is a good idea. It's nicer UX.

Reply:
We should issue a warning if that argument is passed.